cicd/pipelines/understanding-openshift-pipelines.adoc

```diff
@@ -48,3 +48,4 @@ include::modules/op-about-triggers.adoc[leveloffset=+2]
 
 * For information on installing Pipelines, see xref:../../cicd/pipelines/installing-pipelines.adoc#installing-pipelines[Installing OpenShift Pipelines].
 * For more details on creating custom CI/CD solutions, see xref:../../cicd/pipelines/creating-applications-with-cicd-pipelines.adoc#creating-applications-with-cicd-pipelines[Creating applications with CI/CD Pipelines].
+* For more details on re-encrypt TLS termination, see link:https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#re-encryption-termination[Re-encryption Termination].
```

Comment:
The same as in previous doc

Reply:
Will consult with @savitaashture and do the needful. Thanks!

Reply:
i don't have strong opinion here which version to be used

But if latest 4.7 have the same content then i would prefer to use latest version